<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox_address">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>14</number>
      </property>
      <property name="displayIntegerBase">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="initbutton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Initialize the pump and valve&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Zero</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QRadioButton" name="radioButton_pickup">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Pickup</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" rowspan="3">
     <widget class="QSlider" name="poslider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>75</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="maximum">
       <number>3000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_connect">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="plungermovebutton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Move Plunger</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QRadioButton" name="radioButton_absolute">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Absolute</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QRadioButton" name="radioButton_dispense">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Dispense</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="3">
     <widget class="QComboBox" name="comboBox_valve">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Input</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Output</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bypass</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Extra</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_cycle">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Cycle</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_status">
      <property name="text">
       <string>label_status</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_zeroall">
      <property name="text">
       <string>Zero All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionZero">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zero</string>
   </property>
  </action>
  <action name="actionAbsolute_move">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Absolute move</string>
   </property>
  </action>
  <action name="actionPickup_move">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pickup move</string>
   </property>
  </action>
  <action name="actionDispense_move">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dispense move</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>poslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
